src/main/java/com/programmers/engine/model/stack/CalculationStack.java

```diff
@@ -0,0 +1,67 @@
+package com.programmers.engine.model.stack;
+
+import com.programmers.engine.model.confirmation.Confirmation;
+import com.programmers.engine.model.operation.Operator;
+
+import java.util.List;
+import java.util.Stack;
+
+public class CalculationStack {
+
+    private final Stack<Integer> operandStack;
+    private final Stack<String> operatorStack;
+    private final Confirmation confirmation;
+
+    public CalculationStack(Stack<Integer> operandStack, Stack<String> operatorStack, Confirmation confirmation) {
+        this.operandStack = operandStack;
+        this.operatorStack = operatorStack;
+        this.confirmation = confirmation;
+    }
+
+    public Integer popAnswer(List<String> operatorsAndOperands) {
+        for (String element : operatorsAndOperands) {
+            pushOperand(element);
+            pushOperator(element);
+        }
+        calculateRemainStackValue();
+        return operandStack.pop();
+    }
```

Comment:
```suggestion
    public Integer popAnswer(List<String> operatorsAndOperands) {
        calculateRemainStackValue();
        return operandStack.pop();
    }
```

위 코멘트를 적용하면 요 메서드는 정말 pop에만 집중할 수 있겠네요.